daemon/src/db.rs

```diff
@@ -264,23 +264,7 @@ impl Connection {
         C::CtorArgs: Clone + Send + Sync,
     {
         let stream = async_stream::stream! {
-            let mut conn = self.inner.acquire().await?;
-
-            let ids = sqlx::query!(
-                r#"
-                SELECT
-                    uuid as "uuid: model::OrderId"
-                FROM
-                    cfds
-                "#
-            )
-            .fetch_all(&mut *conn)
-            .await?
-            .into_iter()
-            .map(|r| r.uuid);
-
-            drop(conn);
-
+            let ids = self.load_open_cfd_ids().await?;
```

Comment:
We should probably document that callers cannot rely on only open IDs being returned here.